client/src/Editor/ColliderEditor/ColliderEditor.css

```diff
@@ -1,16 +1,21 @@
-.ui_ColliderEditor{
-    background-color: violet;
+.ui_ColliderEditor {
```

Comment:
Is there a coding style checker for css (in order to avoid such cosmetic changes) ?

Reply:
We should to automate style formatting

Reply:
Their seem to already be [eslint and eslint-fix scripts in package.json](https://github.com/VCityTeam/UD-Imuv/blob/master/server/package.json#L8).
Adding a new stage that calls `npm run eslint` in .travis.yml should come at low cost.